test/e2e/common.go

```diff
@@ -20,15 +20,28 @@ import (
 	"context"
 	"fmt"
 	"path/filepath"
+	"strings"
 
 	. "github.com/onsi/ginkgo"
 
+	"github.com/blang/semver"
+	"github.com/onsi/gomega/types"
 	corev1 "k8s.io/api/core/v1"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1alpha3"
 	"sigs.k8s.io/cluster-api/test/framework"
 	"sigs.k8s.io/cluster-api/util"
 )
 
+// Test suite constants for e2e config variables
+const (
+	KubernetesVersionCurrent              = "KUBERNETES_VERSION_CURRENT"
```

Comment:
I'm not convinced by the `Current` Suffix, mostly if you look at this var for tests different from the upgrade test...
I think we should have `KUBERNETES_VERSION` (for all the tests) and 
`KUBERNETES_VERSION_UPGRADE_FROM`, `KUBERNETES_VERSION_UPGRADE_TO`, `ETCD_VERSION_UPGRADE_TO`, `COREDNS_VERSION_UPGRADE_TO` for the upgrade tests